packages/components/src/Chips/InternalChipDismissible/hooks/useInternalChipDismissibleInput.tsx

```diff
@@ -111,7 +111,16 @@ export function useInternalChipDismissibleInput({
       };
 
       if (searchValue.length === 0) {
-        callbacks.Backspace = () => onEmptyBackspace();
+        callbacks.Backspace = () => {
+          // If there's no text left to delete,
+          // and delete is pressed again, focus on a chip instead.
+          const target = computed.inputRef.current?.previousElementSibling;
+          if (target instanceof HTMLElement) {
+            target.focus();
+          } else {
+            onEmptyBackspace();
+          }
+        };
```

Comment:
Hmm, you're right!  I think `onEmptyBackspace()` can be removed, because right now we put the focus on the chip now when there is an empty input.  Would just having the logic in the `handleKeyDown` function you've highlighted here make sense?

I think it would because the functionality of trying to delete something is related to the input right?

Reply:
Cool

![](https://i.imgflip.com/1nhe2n.gif)

Reply:
This commit removes everything related to that functionality: https://github.com/GetJobber/atlantis/pull/872/commits/913c20e21c6e79cc8821ff0a5eda00329d812072